### ISSUE TITLE
Add funder

### ISSUE THREAD

**Issue by ddeboer** [netwerk-digitaal-erfgoed/requirements-datasets#54]
Does it make sense to add a `schema:funder` property?

I got this idea from reading https://cacm.acm.org/magazines/2021/12/256932-datasheets-for-datasets/fulltext.

**Comment by coret** [comment]
Suggested range: `schema:Organization` ?

**Comment by ddeboer** [comment]
Yes, that would make most sense. The property would then be used to give funding organisations more visibility, also in client apps built with Register data, maybe to show some information ‘this collection was made possible by …’.
